features/api.feature

```diff
@@ -146,16 +146,16 @@ Feature: DrupalContext
     | name      |
     | Tag one   |
     | Tag two   |
-    | Tag three |
+    | Tag,three |
     | Tag four  |
     And "article" content:
-    | title           | body             | promote | field_tags                  |
-    | Article by Joe  | PLACEHOLDER BODY |       1 | Tag one, Tag two, Tag three |
-    | Article by Mike | PLACEHOLDER BODY |       1 | Tag four                    |
+    | title           | body             | promote | field_tags                    |
+    | Article by Joe  | PLACEHOLDER BODY |       1 | Tag one, Tag two, "Tag,three" |
```

Comment:
Put a space behind the comma to make this a real example. It would also be clearer if we rename `Tag, three` to `Tag, containing a comma`.